IyiKuyoro/portfolio
# Ft homepage articles 164100843

#### What does this PR do?
This PR adds the articles group component to the homepage

#### Description of Task to be completed?
- [x] Build the home page
- [x] Add the articles group component

#### How should this be manually tested?
- Check out `ft-homepage-articles-164100843`
- Build the server and lunch the home page

#### What are the relevant pivotal tracker stories?
#164100843
